docs/source/installation.mdx

```diff
@@ -16,183 +16,189 @@ limitations under the License.
-``~/.cache/huggingface/transformers/``.
+1. Shell environment variable (default): `TRANSFORMERS_CACHE`.
+2. Shell environment variable: `HF_HOME`.
+3. Shell environment variable: `XDG_CACHE_HOME` + `/huggingface/`.
 
-**Note:** If you have set a shell environment variable for one of the predecessors of this library
-(``PYTORCH_TRANSFORMERS_CACHE`` or ``PYTORCH_PRETRAINED_BERT_CACHE``), those will be used if there is no shell
-environment variable for ``TRANSFORMERS_CACHE``.
+<Tip>
+🤗 Transformers will use the shell environment variables `PYTORCH_TRANSFORMERS_CACHE` or `PYTORCH_PRETRAINED_BERT_CACHE` if you are coming from an earlier iteration of this library unless you specify the shell environment variable `TRANSFORMERS_CACHE`.
+</Tip>
 
-### Offline mode
+## Offline mode
 
-It's possible to run 🤗 Transformers in a firewalled or a no-network environment.
+🤗 Transformers is able to run in a firewalled or offline environment by only using local files. Set the environment variable `TRANSFORMERS_OFFLINE=1` to enable this behavior.
 
-Setting environment variable `TRANSFORMERS_OFFLINE=1` will tell 🤗 Transformers to use local files only and will not try to look things up.
+<Tip>
+Add 🤗 [Datasets](https://huggingface.co/docs/datasets/) to your offline training workflow by setting the environment variable `HF_DATASETS_OFFLINE=1`.
+</Tip>
 
-Most likely you may want to couple this with `HF_DATASETS_OFFLINE=1` that performs the same for 🤗 Datasets if you're using the latter.
+For example, you would typically run a program on a normal network firewalled to external instances with the following command:
 
-Here is an example of how this can be used on a filesystem that is shared between a normally networked and a firewalled to the external world instances.
-
-On the instance with the normal network run your program which will download and cache models (and optionally datasets if you use 🤗 Datasets). For example:
-
-```
+```bash
 python examples/pytorch/translation/run_translation.py --model_name_or_path t5-small --dataset_name wmt16 --dataset_config ro-en ...
 ```
 
-and then with the same filesystem you can now run the same program on a firewalled instance:
-```
+Run this same program in an offline instance with:
+
+```bash
 HF_DATASETS_OFFLINE=1 TRANSFORMERS_OFFLINE=1 \
 python examples/pytorch/translation/run_translation.py --model_name_or_path t5-small --dataset_name wmt16 --dataset_config ro-en ...
 ```
-and it should succeed without any hanging waiting to timeout.
 
-#### Fetching models and tokenizers to use offline
+The script should now run without hanging or waiting to timeout because it knows it should only look for local files.
+
+### Fetch models and tokenizers to use offline
+
+Another option for using 🤗 Transformers offline is to download the files ahead of time, and then point to their local path when you need to use them offline. There are two ways to do this:
```

Comment:
A third option is to use `from_pretrained` first, then `save_pretrained` and have the future `from_pretrained` calls point to the directory specified in the `save_pretrained`

Reply:
Awesome, I'll include this as well!